sdk/storage/storage-blob/review/generated/src/models/index.d.ts

```diff
@@ -0,0 +1,6713 @@
+/// <reference types="node" />
+import { ServiceClientOptions } from "@azure/core-http";
+import * as coreHttp from "@azure/core-http";
+/**
+ * Key information
+ */
+export interface KeyInfo {
+    /**
+     * The date-time the key is active in ISO 8601 UTC time
+     */
+    start: string;
+    /**
+     * The date-time the key expires in ISO 8601 UTC time
+     */
+    expiry: string;
+}
+/**
+ * A user delegation key
+ */
+export interface UserDelegationKey {
+    /**
+     * The Azure Active Directory object ID in GUID format.
+     */
+    signedOid: string;
+    /**
+     * The Azure Active Directory tenant ID in GUID format
+     */
+    signedTid: string;
+    /**
+     * The date-time the key is active
+     * **NOTE: This entity will be treated as a string instead of a Date because the API can
+     * potentially deal with a higher precision value than what is supported by JavaScript.**
+     */
+    signedStart: string;
+    /**
+     * The date-time the key expires
+     * **NOTE: This entity will be treated as a string instead of a Date because the API can
+     * potentially deal with a higher precision value than what is supported by JavaScript.**
+     */
+    signedExpiry: string;
+    /**
+     * Abbreviation of the Azure Storage service that accepts the key
+     */
+    signedService: string;
+    /**
+     * The service version that created the key
+     */
+    signedVersion: string;
+    /**
+     * The key as a base64 string
+     */
+    value: string;
+}
+/**
+ * An interface representing StorageError.
+ */
+export interface StorageError {
+    message?: string;
+}
+/**
+ * The service error response object.
+ */
+export interface DataLakeStorageErrorError {
```

Comment:
Also, this type name is strange, with these 2x "Error"

Reply:
The `DataLake` likely shouldn't be here. I don't think we use it. it could be an error in the swagger json.

Reply:
Will hide it for next release (#5457)